src/com/facebook/buck/android/AndroidAarDescription.java

```diff
@@ -179,7 +179,8 @@ public Arg createUnpopulatedConstructorArg() {
         new BuildTargetSourcePath(assembleAssetsDirectories.getBuildTarget()),
         /* assetsSrcs */ ImmutableSortedSet.<Path>of(),
         new BuildTargetSourcePath(manifest.getBuildTarget()),
-        /* hasWhitelistedStrings */ false);
+        /* hasWhitelistedStrings */ false,
+        /* includeTransitiveDeps */ false);
```

Comment:
Let's take a `AndroidLibraryGraphEnhancer.ResourceDependencyMode` instead of a boolean here